backstop/tests/information-panel.html

```diff
@@ -1261,127 +1261,148 @@ <h1>Information panel</h1>
               </button>
             </div>
           </div>
+          <div
+            class="iui-scroll-shadow-wrapper"
+            style='overflow: hidden; position: relative;'
+          >
+            <div
+              class="iui-scroll-shadow iui-shadow-top"
+              style="position: absolute;
+    width: 100%;
+    height: 11px;
+    top: 0;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.35), transparent);
+    opacity: 0;"
```

Comment:
@FlyersPh9 Not sure how to get the `iui-scroll-shadow iui-shadow-top` mixin to work here